Create 2020-02-07-DataPlant.md (#394)
usegalaxy-eu/website: _posts/2020-02-07-DataPlant.md

---
site: freiburg

> title: Galaxy provides support in a NFDI

Reviewer: as a question? or

We applied to support the NFDI.

> title: Galaxy provides support in a NFDI

Reviewer 2: ```suggestion
title: Galaxy provides support as an NFDI
```

---

Germany wants to build a national research data infrastructure [NFDI](https://www.dfg.de/foerderung/programme/nfdi)
which can systematically manage scientific and research data, provide long-term data storage,
backup and accessibility, and network the data both nationally and internationally.

> The University of Freiburg applied as lead with the "DataPLANT" project to connect fundamental plant researchers in

Reviewer 2: ```suggestion
The University of Freiburg applied as a lead with the "DataPLANT" project to connect plant researchers in
```

Germany and to provide a suitable infrastructure for data analysis. 

Five representatives from the DataPLANT consortium met in the beginning of December 2019 in Bonn to present and defend

> the proposal in front of 22 reviewers and DFG members. From our Team, Anika joined to give insights about

Reviewer 2: ```suggestion
the proposal in front of 22 reviewers and DFG members. Anika represented our team, providing insights about
```

the data management capabilities of Galaxy and Training.
The other representatives were Dirk von Suchodoletz (RZ, University of Freiburg),
Björn Usadel (Forschungszentrum Jülich), Jens Krüger (University of Tübingen),
and Timo Mühlhaus (University of Kaiserslautern).

> Now we received a very positive feedback from the reviewers. So, let's keep fingers crossed.

Reviewer: maybe we just write about this meeting in december and not mention the review?

> Now we received a very positive feedback from the reviewers. So, let's keep fingers crossed.

Reviewer 2: ```suggestion
Since then, we received very positive feedback from the reviewers. So, let's keep our fingers crossed.
```
